tests/foreman/ui/test_settings.py

```diff
@@ -96,125 +70,132 @@ def add_content_views_to_composite(composite_cv, org, repo):
     """
-    property_names = ['entries_per_page', 'foreman_url']
+    property_name = setting_update.name
     with session:
-        for property_name in property_names:
-            set_original_property_value(property_name)
-            for param_value in invalid_settings_values():
-                with raises(AssertionError) as context:
-                    session.settings.update('name = {}'.format(property_name), param_value)
-                assert is_valid_error_message(str(context.value))
+        invalid_value = [invalid_value for invalid_value in invalid_settings_values()][0]
+        with raises(AssertionError) as context:
+            session.settings.update(f'name = {property_name}', invalid_value)
+            assert is_valid_error_message(str(context.value))
 
 
 @run_in_one_thread
 @tier2
-def test_positive_selectors(session):
-    """"Testing input for selectors: dropdown, text area, input box
+@pytest.mark.parametrize(
+    'setting_update',
+    ['host_dmi_uuid_duplicates', 'register_hostname_fact', 'content_view_solve_dependencies'],
+    indirect=True,
+)
+def test_positive_host_dmi_uuid_duplicates(session, setting_update):
+    """"check the setting host_dmi_uuid_duplicates, register_hostname_fact &
+    content_view_solve_dependencies  value update.
 
     :id: 529ddd3a-1271-4043-9006-eac436b08b11
 
+    :parametrized: yes
+
     :expectedresults: Successfully add value to different selectors
 
     :CaseImportance: High
     """
-    uuid_input = "[ {} ]".format(gen_string("alpha"))
-    random_input = gen_string('alpha')
-    uuid_duplicate_prop = ('host_dmi_uuid_duplicates', uuid_input)
-    reg_host_prop = ('register_hostname_fact', random_input)
-    dep_solve_prop = ('content_view_solve_dependencies', 'Yes')
+    property_dict = {"content_view_solve_dependencies": "Yes", "host_dmi_uuid_duplicates":
+                     f'[ {gen_string("alpha")} ]', "register_hostname_fact": gen_string('alpha')}
 
+    property_name = setting_update.name
     with session:
-        for setting_attr, value in [uuid_duplicate_prop, dep_solve_prop, reg_host_prop]:
-            # Stores original values for each type of selectors
-            original_val = entities.Setting().search(
-                query={'search': 'name={}'.format(setting_attr)}
-            )[0]
-            # Update to new value and resets value to back to old state
-            try:
-                session.settings.update('name = {}'.format(setting_attr), value)
-                result = session.settings.read('name = {}'.format(setting_attr))
-                assert result['table'][0]['Value'] == value
-            finally:
-                setting_cleanup(setting_attr, str(original_val.default))
+        session.settings.update(f'name = {property_name}', property_dict[setting_update.name])
+        result = session.settings.read(f'name = {property_name}')
+        assert result['table'][0]['Value'] ==  property_dict[setting_update.name]
```

Comment:
There are two spaces after `==`, and black doesn't like that either.

Reply:
fixed.